package.json

```diff
@@ -15,6 +15,7 @@
     "babel-loader": "^5.3.2",
     "babel-plugin-espower": "^1.0.0",
     "babel-runtime": "^5.8.20",
+    "classnames": "^2.2.5",
```

Comment:
Ok. Anyway you could use stateless components and `const` except of `let` in properties destructuring (this variables are never reassigned).